src/Components/Credits/LoanAmortizationSimulation/LoanAmortizationSimulationForm.tsx

```diff
@@ -0,0 +1,148 @@
+import { ChangeEvent, MouseEvent, useEffect, useState } from 'react'
+import '../../../Styles/Form.sass'
+import { loanAmortizationSimulationSelectProperty } from '../../../assets'
+import { FormButton, Input, SelectInput, parseInputState } from '../../Global'
+
+interface LoanAmortizationSimulationFormProps {
+  loanValue: number
+  duration: number
+  marginOfTheBank: number
+  commisionFee: number
+  interestAccrualMethod: string
+  doesTheBankChargeACommission: string
+  setCommisionFee: (value: number) => void
+  setLoanRepaymentMethod: (value: string) => void
+  handleSetLoanValue: (value: ChangeEvent<HTMLInputElement>) => void
+  handleSetDuration: (value: ChangeEvent<HTMLInputElement>) => void
+  handleSetMarginOfTheBank: (value: ChangeEvent<HTMLInputElement>) => void
+  handleSetCommisionFee: (value: ChangeEvent<HTMLInputElement>) => void
+  handleSetOptionDuration: (value: ChangeEvent<HTMLInputElement>) => void
+  handleSetPaymentPeriodOfInstallment: (
+    value: ChangeEvent<HTMLInputElement>
+  ) => void
+  handleSetInterestAccrualMethod: (value: ChangeEvent<HTMLInputElement>) => void
+  handleSetDoesTheBankChargeACommission: (
+    value: ChangeEvent<HTMLInputElement>
+  ) => void
+  handleSetLoanRepaymentMethod: (value: ChangeEvent<HTMLInputElement>) => void
+  calculate: (value: MouseEvent<HTMLButtonElement>) => void
+}
+
+const LoanAmortizationSimulationForm = ({
+  loanValue,
+  duration,
+  marginOfTheBank,
+  commisionFee,
+  interestAccrualMethod,
+  doesTheBankChargeACommission,
+  setCommisionFee,
+  setLoanRepaymentMethod,
+  handleSetLoanValue,
+  handleSetDuration,
+  handleSetMarginOfTheBank,
+  handleSetCommisionFee,
+  handleSetOptionDuration,
+  handleSetPaymentPeriodOfInstallment,
+  handleSetInterestAccrualMethod,
+  handleSetDoesTheBankChargeACommission,
+  handleSetLoanRepaymentMethod,
+  calculate,
+}: LoanAmortizationSimulationFormProps) => {
+  useEffect(() => {
+    setCommisionFee(0)
+  }, [doesTheBankChargeACommission])
+
+  const [prevInterestAccrualMethod, setPrevInterestAccrualMethod] = useState('')
+
+  useEffect(() => {
+    if (interestAccrualMethod === 'InterestPaidInAdvance') {
+      setLoanRepaymentMethod('DecreasingInstallments')
+    } else if (
+      interestAccrualMethod !== 'InterestPaidInAdvance' &&
+      prevInterestAccrualMethod === 'InterestPaidInAdvance'
+    ) {
+      setLoanRepaymentMethod('')
+    }
+
+    setPrevInterestAccrualMethod(interestAccrualMethod)
+  }, [interestAccrualMethod, prevInterestAccrualMethod])
+
+  const {
+    propertyForSetOptionDuration,
+    propertyForSetPaymentPeriodOfInstallment,
+    propertyForSetInterestAccrualMethod,
+    propertyForSetDoesTheBankChargeACommission,
+    propertyForSetLoanRepaymentMethod,
+  } = loanAmortizationSimulationSelectProperty
+
+  const checkDoesTheBankChargeACommission = doesTheBankChargeACommission ===
```

Comment:
it's variable for component but it means that it does something, call it using `Input` maybe